src/components/NewWorkspaceModal.js

```diff
@@ -63,14 +63,15 @@ const ariaInvalidBillingAccountMsg = (invalidBillingAccount) => {
 
 const NewWorkspaceModal = withDisplayName(
   'NewWorkspaceModal',
-  ({ cloneWorkspace, onSuccess, onDismiss, customMessage, requiredAuthDomain, title, buttonText }) => {
+  ({ cloneWorkspace, onSuccess, onDismiss, customMessage, requiredAuthDomain, requireEnhancedBucketLogging, title, buttonText }) => {
     // State
     const [billingProjects, setBillingProjects] = useState();
     const [allGroups, setAllGroups] = useState();
     const [name, setName] = useState(cloneWorkspace ? `${cloneWorkspace.workspace.name} copy` : '');
     const [namespace, setNamespace] = useState(cloneWorkspace ? cloneWorkspace.workspace.namespace : undefined);
     const [description, setDescription] = useState(cloneWorkspace ? cloneWorkspace.workspace.attributes.description : '');
     const [groups, setGroups] = useState([]);
+    const [enhancedBucketLogging, setEnhancedBucketLogging] = useState(requireEnhancedBucketLogging);
```

Comment:
To fix the a11y issue about the checked state not being set when initially rendered, make this !!requireEnhancedBucketLogging (so it is false instead of undefined).